moveit_planners/chomp/chomp_optimizer_adapter/src/chomp_optimizer_adapter.cpp

```diff
@@ -213,4 +215,4 @@ class OptimizerAdapter : public planning_request_adapter::PlanningRequestAdapter
 };
 }  // namespace chomp
 
-CLASS_LOADER_REGISTER_CLASS(chomp::OptimizerAdapter, planning_request_adapter::PlanningRequestAdapter);
+CLASS_LOADER_REGISTER_CLASS(chomp::OptimizerAdapter, planning_request_adapter::PlanningRequestAdapter)
```

Comment:
This should better use `PLUGINLIB_EXPORT_CLASS`

Reply:
Fixed.